testsuite/performance/tests/pom.xml

```diff
@@ -253,6 +255,13 @@
                     </execution>
                 </executions>
             </plugin>                    
+            <plugin>
```

Comment:
Why is this plugin added here?

Reply:
Most of the executions of `exec-maven-plugin` in this pom use: `<workingDirectory>${project.basedir}/..</workingDirectory>` because they run docker-compose or scripts in the parent directory. My intention was to move this configuration to the common plugin-level configuration and remove duplicate entries from the execution-level configruations.

But it seems I didn't finish the job as I only removed the duplicate entries from within the import-dump profile. I need to remove the rest.